dcm4che-tool/dcm4che-tool-stowrs/pom.xml

```diff
@@ -50,7 +50,12 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-log4j12</artifactId>
     </dependency>
-        <dependency>
+    <dependency>
+      <groupId>log4j</groupId>
```

Comment:
No need to pull in log4j explicitly. The dependency above to slf4j-log4j12 should already do it.